actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -22,13 +22,25 @@ def render_action_text_attachments(content)
       content.render_attachments do |attachment|
         unless attachment.in?(content.gallery_attachments)
           attachment.node.tap do |node|
-            node.inner_html = render(attachment, in_gallery: false).chomp
+            partial = if attachment.respond_to?(:to_attachable_partial_path)
+              attachment.to_attachable_partial_path
+            else
+              attachment
+            end
+
+            node.inner_html = render(partial: partial, object: attachment, locals: { in_gallery: false }).chomp
```

Comment:
@georgeclaghorn do you think that pairing `partial: ..., object: ...` is sufficient enough? Without specifying `as: ...`, the template-local variable will reflect the name of the partial.

For example, `people/_attachable.html.erb` results in an `attachable` local variable, while `people/_my_weird_workaround_name.html.erb` results in a `my_weird_workaround_name` local variable.

At this point in rendering, are we confident that the `attachment` responds to [model_name](https://edgeapi.rubyonrails.org/classes/ActiveModel/Naming.html#method-i-model_name)? If so, we could pass along `as: attachment.model_name.element` to ensure that `people/_my_weird_workaround.html.erb` can reference the `Person` instance via a `person` local variable.

Reply:
> At this point in rendering, are we confident that the `attachment` responds to [model_name](https://edgeapi.rubyonrails.org/classes/ActiveModel/Naming.html#method-i-model_name)? If so, we could pass along `as: attachment.model_name.element` to ensure that `people/_my_weird_workaround.html.erb` can reference the `Person` instance via a `person` local variable.

I think we are, and I would prefer that.